lib/ansible/modules/network/aci/aci_access_port_to_interface_policy_leaf_profile.py

```diff
@@ -60,6 +60,12 @@
     description:
     - The name of the fabric access policy group to be associated with the leaf interface profile interface selector.
     aliases: [ policy_group_name ]
+  policy_group_type:
```

Comment:
So in **aci_static_binding_to_epg** this parameter was named **interface_type** and we prefer to keep the parameter names consistent between modules.
```yaml
  interface_type:
    description:
    - The type of interface for the static EPG deployement.
    choices: [ fex, port_channel, switch_port, vpc ]
    default: switch_port
```